config/templates/operator/vacuum.yaml

```diff
@@ -0,0 +1,129 @@
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: "kubearchive-cluster-vacuum"
+  namespace: kubearchive
+  labels:
+    app.kubernetes.io/name: "kubearchive-operator"
+    app.kubernetes.io/component: operator
```

Comment:
It does not matter that much, but here we could use `vacuum`.

Reply:
No, that's a good point.  I've changed all the resources in that file.